dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,38 @@
+import javax.crypto.Cipher;
+import javax.crypto.NoSuchPaddingException;
+import java.security.NoSuchAlgorithmException;
+
+public class Demo {
+    public static void main(String[] args) throws Exception {
+        String rpository = "test";
+        int wek = 4;
+        String day;
+    
+        Cipher c = Cipher.getInstance("DESede/ECB/PKCS5Padding");
+        
+        switch (wek) {
```

Comment:
RSPEC-131: Switch without default case
<!-- lookout footnote separator -->
_Comment made by the analyzer sonar-check._